src/sonic-yang-models/yang-models/sonic-acl.yang

```diff
@@ -11,12 +11,10 @@ module sonic-acl {
 
 	import sonic-types {
 		prefix stypes;
-		revision-date 2019-07-01;
```

Comment:
@maheshwari-mayank What's the motivation to remove `revision-date`?
Please elaborate in the PR's description.

Also, how does it correlate with RFC 7950?
https://tools.ietf.org/html/rfc7950#section-7.1.5

```
When the optional "revision-date" substatement is present, any
typedef, grouping, extension, feature, and identity referenced by
definitions in the local module are taken from the specified revision
of the imported module.  It is an error if the specified revision of
the imported module does not exist.  If no "revision-date"
substatement is present, it is undefined from which revision of the
module they are taken.
```

Reply:
@praveen-li / @maheshwari-mayank opinion?

Reply:
'revision-date' within import statement is required if we maintain multiple versions of same yang. As SONiC yang is a model of DB table schema, we do not need multiple versions of same yang on switch.

Reply:
In Yang work group, nobody could very well explained the use case of revision date on Sonic switch. So I am good if we are to remove it at all places. Let me know if we think otherwise.